src/Manager/AlbumManager.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types = 1);
+
+namespace App\Manager;
+
+use App\Entity\Album;
+use App\Repository\AlbumRepository;
+use App\Repository\ArtistRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use Doctrine\Persistence\ObjectManager;
+
+class AlbumManager
+{
+    private ObjectManager $entityManager;
+    private AlbumRepository $albumRepository;
+    private ArtistRepository $artistRepository;
+    private ArtistManager $artistManager;
+
+    public function __construct(
+        ManagerRegistry $managerRegistry,
+        AlbumRepository $albumRepository,
+        ArtistRepository $artistRepository,
+        ArtistManager $artistManager
+    ) {
```

Comment:
If you are in SF6, then you're in PHP8 and so you can do something brilliant like this : 
```php
    private ObjectManager $entityManager;

     public function __construct(
        ManagerRegistry $managerRegistry,
        private AlbumRepository $albumRepository,
        private ArtistRepository $artistRepository,
        private ArtistManager $artistManager
    ) {
         $this->entityManager    = $managerRegistry->getManager();
    }
```
instead of declaring and hydrating your dependencies 
![image](https://user-images.githubusercontent.com/2346468/164997439-be26c600-9dd1-4be5-9ba1-efd7864cf345.png)

Reply:
All that code I disciplined myself to write for years does no longer need to be written I don't know if I rejoice or cry 😁